api/envoy/extensions/http/header_formatters/preserve_case/v3/preserve_case.proto

```diff
@@ -17,7 +17,21 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // See the :ref:`header casing <config_http_conn_man_header_casing>` configuration guide for more
 // information.
 message PreserveCaseFormatterConfig {
+  enum FormatterTypeOnUnknownHeaders {
```

Comment:
If you add a PGV validation for the enum you can guarantee it is one of these options and then the error handling in the switch statement can be simpler.

Reply:
done